src/storage/src/hummock/store/version.rs

```diff
@@ -108,12 +105,57 @@ pub struct HummockReadVersion {
     committed: CommittedVersion,
 }
 
-#[expect(unused_variables)]
+impl Default for HummockReadVersion {
+    fn default() -> Self {
```

Comment:
I think we should not allow `HummockReadVersion` to hold an invalid default `HummockVersion`. Instead we may add a `new` method for it and pass a valid `PinnedVersion` to it.

Reply:
Thansk, we build the `ReadVersion` with `PinnedVersion ` which is provided by `LocalVersionManager`. To ensure the  Validation of input `PinnedVersion`, add a assert condition to  guarantee correct initialization order